cd-menu/cd-menu.js

```diff
@@ -134,7 +135,24 @@ window.cdMenu = function (options) {
         loginRegister.style.display = 'block';
       });
     }
-
+    function canSeeELearningModule (userDojos) {
```

Comment:
We could, but until we don't have more use case, there is no point in generalizing? @DanielBrierton